presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -51,6 +51,9 @@
     @VisibleForTesting
     static final String SPILLER_SPILL_PATH = "experimental.spiller-spill-path";
 
+    private double cpuCostWeight = 75;
```

Comment:
it would be nice to have them in session as well

Reply:
yes, when this is actually useful for someone